integration-tests/tests/src/tests/import-app-util.ts

```diff
@@ -10,7 +10,7 @@ describe.skipIf(environment.missingServer, "importApp utility", function() {
     const app = await importApp("simple");
     expect(app).instanceOf(App);
     expect(app.id.startsWith("simple")).equals(true);
-  });
+  }).timeout(10000);
```

Comment:
Anyway we can get rid of the timeout?  This overtime will really start slowing the CI down.

Reply:
Again, this is not waiting 10s - it just ensures that the test won't fail if it takes more than 2s.